assets/_locales/en/messages.json

```diff
@@ -1997,5 +1997,21 @@
   "recommended":  {
     "message": "Recommended",
     "description": "required"
+  },
+  "add_wallet_error_message": {
+    "message": "Please enter a recovery phrase or key file and enter the password to your current wallet before continuing.",
+    "description": "Error message if recovery section is left blank"
+  },
+  "empty_password_error_message": {
+    "message": "Please enter the password to your current wallet before continuing. Not the password to the wallet you are importing.",
```

Comment:
Not a fan of this copy. Maybe we could signal somewhere that the password is Arconnect-bound and *not* wallet-bound